torch/__init__.py

```diff
@@ -1065,6 +1066,22 @@ def set_float32_matmul_precision(precision: str) -> None:
     """
     _C._set_float32_matmul_precision(precision)
 
+def get_matmul_autotune() -> builtins.bool:
+    r"""Returns the current value of the matrix multiplication autotuning flag. Refer to
+    :func:`torch.set_matmul_autotune` documentation for more details.
+    """
+    return _C._get_matmul_autotune()
+
+def set_matmul_autotune(enabled: bool) -> None:
+    r"""Sets the autotuning flag for matrix multiplications.
+
+    Certain matmuls can be folded into bmm (batched matrix multiply), and by default the decision
```

Comment:
If one of the arguments of batched matrix multiplication is N-dimensions with N >= 3 and the other argument is 2-dimensional then "should_fold" decides whether to fold the N-dimensional argument into 2-dimensional and call a single matrix multiplication (`torch.mm`).